### ISSUE TITLE
Script fails after phase 4 - instance not accessible through SSH

### ISSUE THREAD

**Issue by MrKnarfy** [sormy/gentoo-ami-builder#13]
When attempting to build a default gentoo profile on m5zn.metal, m5zn.6xlarge and c5.2xlarge, the instance apparently becomes inaccessible over ssh.
I shall try again and will add the output to this issue.

Cheers.
Frank.

PS: this was all in us-west-2. I will now run the command line as suggested in the documentation.

**Comment by sormy** [comment]
Thanks for reporting. Could you please take a look what is on serial console?

**Comment by sormy** [comment]
I can confirm that instance doesn't boot healthy after phase 4. Will take a look.

**Comment by sormy** [comment]
```sh
[H[J[1;1H[H[J[1;1H[    0.000000] Linux version 5.10.61-gentoo-x86_64 (root@ip-172-31-17-134.ec2.internal) (x86_64-pc-linux-gnu-gcc (Gentoo 10.3.0-r2 p3) 10.3.0, GNU ld (Gentoo 2.37_p1 p0) 2.37) #1 SMP Mon Oct 4 00:00:02 -00 2021
[    0.000000] Command line: BOOT_IMAGE=/boot/vmlinuz-5.10.61-gentoo-x86_64 root=UUID=4635af18-d23d-4770-a867-3d3d32b283da net.ifnames=0 ro console=tty0 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0 nvme_core.io_timeout=4294967295 rd.emergency=poweroff rd.shell=0
[    0.000000] x86/fpu: Supporting XSAVE feature 0x001: 'x87 floating point registers'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x002: 'SSE registers'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x004: 'AVX registers'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x008: 'MPX bounds registers'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x010: 'MPX CSR'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x020: 'AVX-512 opmask'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x040: 'AVX-512 Hi256'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x080: 'AVX-512 ZMM_Hi256'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x200: 'Protection Keys User registers'
[    0.000000] x86/fpu: xstate_offset[2]:  576, xstate_sizes[2]:  256
[    0.000000] x86/fpu: xstate_offset[3]:  832, xstate_sizes[3]:   64
[    0.000000] x86/fpu: xstate_offset[4]:  896, xstate_sizes[4]:   64
[    0.000000] x86/fpu: xstate_offset[5]:  960, xstate_sizes[5]:   64
[    0.000000] x86/fpu: xstate_offset[6]: 1024, xstate_sizes[6]:  512
[    0.000000] x86/fpu: xstate_offset[7]: 1536, xstate_sizes[7]: 1024
[    0.000000] x86/fpu: xstate_offset[9]: 2560, xstate_sizes[9]:    8
[    0.000000] x86/fpu: Enabled xstate features 0x2ff, context size is 2568 bytes, using 'compacted' format.
[    0.000000] BIOS-provided physical RAM map:
[    0.000000] BIOS-e820: [mem 0x0000000000000000-0x000000000009fbff] usable
[    0.000000] BIOS-e820: [mem 0x000000000009fc00-0x000000000009ffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000000f0000-0x00000000000fffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000100000-0x00000000bffe8fff] usable
[    0.000000] BIOS-e820: [mem 0x00000000bffe9000-0x00000000bfffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000e0000000-0x00000000e03fffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fffc0000-0x00000000ffffffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000100000000-0x000000041b7fffff] usable
[    0.000000] BIOS-e820: [mem 0x000000041b800000-0x000000043fffffff] reserved
[    0.000000] NX (Execute Disable) protection: active
[    0.000000] SMBIOS 2.7 present.
[    0.000000] DMI: Amazon EC2 c5.2xlarge/, BIOS 1.0 10/16/2017
[    0.000000] Hypervisor detected: KVM
[    0.000000] kvm-clock: Using msrs 4b564d01 and 4b564d00
[    0.000000] kvm-clock: cpu 0, msr 3401001, primary cpu clock
[    0.000000] kvm-clock: using sched offset of 1726722875 cycles
[    0.000002] clocksource: kvm-clock: mask: 0xffffffffffffffff max_cycles: 0x1cd42e4dffb, max_idle_ns: 881590591483 ns
[    0.000004] tsc: Detected 2999.998 MHz processor
[    0.000455] last_pfn = 0x41b800 max_arch_pfn = 0x400000000
[    0.000504] x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WP  UC- WT  
[    0.000510] last_pfn = 0xbffe9 max_arch_pfn = 0x400000000
[    0.006860] check: Scanning 1 areas for low memory corruption
[    0.006939] Using GB pages for direct mapping
[    0.007184] RAMDISK: [mem 0x370be000-0x37856fff]
[    0.007190] ACPI: Early table checksum verification disabled
[    0.007195] ACPI: RSDP 0x00000000000F8F40 000014 (v00 AMAZON)
[    0.007201] ACPI: RSDT 0x00000000BFFEE080 000044 (v01 AMAZON AMZNRSDT 00000001 AMZN 00000001)
[    0.007205] ACPI: FACP 0x00000000BFFEFF80 000074 (v01 AMAZON AMZNFACP 00000001 AMZN 00000001)
[    0.007210] ACPI: DSDT 0x00000000BFFEE0D0 0010E9 (v01 AMAZON AMZNDSDT 00000001 AMZN 00000001)
[    0.007213] ACPI: FACS 0x00000000BFFEFF40 000040
[    0.007215] ACPI: SSDT 0x00000000BFFEF480 000AC0 (v01 AMAZON AMZNSSDT 00000001 AMZN 00000001)
[    0.007218] ACPI: APIC 0x00000000BFFEF360 0000A6 (v01 AMAZON AMZNAPIC 00000001 AMZN 00000001)
[    0.007221] ACPI: SRAT 0x00000000BFFEF260 000100 (v01 AMAZON AMZNSRAT 00000001 AMZN 00000001)
[    0.007224] ACPI: SLIT 0x00000000BFFEF1F0 00006C (v01 AMAZON AMZNSLIT 00000001 AMZN 00000001)
[    0.007226] ACPI: WAET 0x00000000BFFEF1C0 000028 (v01 AMAZON AMZNWAET 00000001 AMZN 00000001)
[    0.007230] ACPI: HPET 0x00000000000C9000 000038 (v01 AMAZON AMZNHPET 00000001 AMZN 00000001)
[    0.007233] ACPI: SSDT 0x00000000000C9040 00007B (v01 AMAZON AMZNSSDT 00000001 AMZN 00000001)
[    0.007236] ACPI: Reserving FACP table memory at [mem 0xbffeff80-0xbffefff3]
[    0.007237] ACPI: Reserving DSDT table memory at [mem 0xbffee0d0-0xbffef1b8]
[    0.007238] ACPI: Reserving FACS table memory at [mem 0xbffeff40-0xbffeff7f]
[    0.007238] ACPI: Reserving SSDT table memory at [mem 0xbffef480-0xbffeff3f]
[    0.007239] ACPI: Reserving APIC table memory at [mem 0xbffef360-0xbffef405]
[    0.007240] ACPI: Reserving SRAT table memory at [mem 0xbffef260-0xbffef35f]
[    0.007241] ACPI: Reserving SLIT table memory at [mem 0xbffef1f0-0xbffef25b]
[    0.007242] ACPI: Reserving WAET table memory at [mem 0xbffef1c0-0xbffef1e7]
[    0.007243] ACPI: Reserving HPET table memory at [mem 0xc9000-0xc9037]
[    0.007244] ACPI: Reserving SSDT table memory at [mem 0xc9040-0xc90ba]
[    0.007289] SRAT: PXM 0 -> APIC 0x00 -> Node 0
[    0.007291] SRAT: PXM 0 -> APIC 0x01 -> Node 0
[    0.007292] SRAT: PXM 0 -> APIC 0x02 -> Node 0
[    0.007294] SRAT: PXM 0 -> APIC 0x03 -> Node 0
[    0.007295] SRAT: PXM 0 -> APIC 0x04 -> Node 0
[    0.007297] SRAT: PXM 0 -> APIC 0x05 -> Node 0
[    0.007298] SRAT: PXM 0 -> APIC 0x06 -> Node 0
[    0.007300] SRAT: PXM 0 -> APIC 0x07 -> Node 0
[    0.007303] ACPI: SRAT: Node 0 PXM 0 [mem 0x00000000-0xbfffffff]
[    0.007305] ACPI: SRAT: Node 0 PXM 0 [mem 0x100000000-0x43fffffff]
[    0.007314] NUMA: Node 0 [mem 0x00000000-0xbfffffff] + [mem 0x100000000-0x41b7fffff] -> [mem 0x00000000-0x41b7fffff]
[    0.007328] NODE_DATA(0) allocated [mem 0x41b7db000-0x41b7fcfff]
[    0.007798] Zone ranges:
[    0.007799]   DMA      [mem 0x0000000000001000-0x0000000000ffffff]
[    0.007800]   DMA32    [mem 0x0000000001000000-0x00000000ffffffff]
[    0.007802]   Normal   [mem 0x0000000100000000-0x000000041b7fffff]
[    0.007803] Movable zone start for each node
[    0.007806] Early memory node ranges
[    0.007807]   node   0: [mem 0x0000000000001000-0x000000000009efff]
[    0.007808]   node   0: [mem 0x0000000000100000-0x00000000bffe8fff]
[    0.007809]   node   0: [mem 0x0000000100000000-0x000000041b7fffff]
[    0.007810] Initmem setup node 0 [mem 0x0000000000001000-0x000000041b7fffff]
[    0.007864] On node 0, zone DMA: 1 pages in unavailable ranges
[    0.007905] On node 0, zone DMA: 97 pages in unavailable ranges
[    0.015412] On node 0, zone Normal: 23 pages in unavailable ranges
[    0.015576] On node 0, zone Normal: 18432 pages in unavailable ranges
[    0.016109] ACPI: PM-Timer IO Port: 0xb008
[    0.016119] ACPI: LAPIC_NMI (acpi_id[0xff] dfl dfl lint[0x1])
[    0.016157] IOAPIC[0]: apic_id 0, version 17, address 0xfec00000, GSI 0-23
[    0.016159] ACPI: INT_SRC_OVR (bus 0 bus_irq 5 global_irq 5 high level)
[    0.016160] ACPI: INT_SRC_OVR (bus 0 bus_irq 9 global_irq 9 high level)
[    0.016161] ACPI: INT_SRC_OVR (bus 0 bus_irq 10 global_irq 10 high level)
[    0.016162] ACPI: INT_SRC_OVR (bus 0 bus_irq 11 global_irq 11 high level)
[    0.016167] Using ACPI (MADT) for SMP configuration information
[    0.016169] ACPI: HPET id: 0x8086a201 base: 0xfed00000
[    0.016172] TSC deadline timer available
[    0.016173] smpboot: Allowing 8 CPUs, 0 hotplug CPUs
[    0.016192] PM: hibernation: Registered nosave memory: [mem 0x00000000-0x00000fff]
[    0.016194] PM: hibernation: Registered nosave memory: [mem 0x0009f000-0x0009ffff]
[    0.016195] PM: hibernation: Registered nosave memory: [mem 0x000a0000-0x000effff]
[    0.016195] PM: hibernation: Registered nosave memory: [mem 0x000f0000-0x000fffff]
[    0.016197] PM: hibernation: Registered nosave memory: [mem 0xbffe9000-0xbfffffff]
[    0.016198] PM: hibernation: Registered nosave memory: [mem 0xc0000000-0xdfffffff]
[    0.016198] PM: hibernation: Registered nosave memory: [mem 0xe0000000-0xe03fffff]
[    0.016199] PM: hibernation: Registered nosave memory: [mem 0xe0400000-0xfffbffff]
[    0.016200] PM: hibernation: Registered nosave memory: [mem 0xfffc0000-0xffffffff]
[    0.016202] [mem 0xc0000000-0xdfffffff] available for PCI devices
[    0.016203] Booting paravirtualized kernel on KVM
[    0.016205] clocksource: refined-jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645519600211568 ns
[    0.019201] setup_percpu: NR_CPUS:8192 nr_cpumask_bits:8 nr_cpu_ids:8 nr_node_ids:1
[    0.019582] percpu: Embedded 55 pages/cpu s188416 r8192 d28672 u262144
[    0.020026] kvm-guest: stealtime: cpu 0, msr 40bc2d080
[    0.020029] kvm-guest: PV spinlocks disabled, no host support
[    0.020033] Built 1 zonelists, mobility grouping on.  Total pages: 3981458
[    0.020034] Policy zone: Normal
[    0.020036] Kernel command line: BOOT_IMAGE=/boot/vmlinuz-5.10.61-gentoo-x86_64 root=UUID=4635af18-d23d-4770-a867-3d3d32b283da net.ifnames=0 ro console=tty0 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0 nvme_core.io_timeout=4294967295 rd.emergency=poweroff rd.shell=0
[    0.022411] Dentry cache hash table entries: 2097152 (order: 12, 16777216 bytes, linear)
[    0.023570] Inode-cache hash table entries: 1048576 (order: 11, 8388608 bytes, linear)
[    0.023638] mem auto-init: stack:off, heap alloc:off, heap free:off
[    0.042698] Memory: 3163720K/16178716K available (10246K kernel code, 9141K rwdata, 3592K rodata, 2252K init, 7628K bss, 393208K reserved, 0K cma-reserved)
[    0.042706] random: get_random_u64 called from kmem_cache_open+0x20/0x1f0 with crng_init=0
[    0.042751] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=8, Nodes=1
[    0.042757] Kernel/User page tables isolation: enabled
[    0.042771] ftrace: allocating 34607 entries in 136 pages
[    0.053267] ftrace: allocated 136 pages with 2 groups
[    0.053359] rcu: Hierarchical RCU implementation.
[    0.053361] rcu: 	RCU restricting CPUs from NR_CPUS=8192 to nr_cpu_ids=8.
[    0.053362] 	Rude variant of Tasks RCU enabled.
[    0.053363] 	Tracing variant of Tasks RCU enabled.
[    0.053364] rcu: RCU calculated value of scheduler-enlistment delay is 25 jiffies.
[    0.053365] rcu: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=8
[    0.056090] NR_IRQS: 524544, nr_irqs: 488, preallocated irqs: 16
[    0.134208] Console: colour VGA+ 80x25
[    0.631810] printk: console [tty0] enabled
[    0.811022] printk: console [ttyS0] enabled
[    0.814422] ACPI: Core revision 20200925
[    0.817913] clocksource: hpet: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 30580167144 ns
[    0.824850] APIC: Switch to symmetric I/O mode setup
[    0.828802] x2apic enabled
[    0.884649] Switched APIC routing to physical x2apic.
[    0.889388] clocksource: tsc-early: mask: 0xffffffffffffffff max_cycles: 0x2b3e43c8763, max_idle_ns: 440795360101 ns
[    0.896876] Calibrating delay loop (skipped) preset value.. 5999.99 BogoMIPS (lpj=11999992)
[    0.900872] pid_max: default: 32768 minimum: 301
[    0.900872] LSM: Security Framework initializing
[    0.900872] Yama: becoming mindful.
[    0.900872] SELinux:  Initializing.
[    0.900872] Mount-cache hash table entries: 32768 (order: 6, 262144 bytes, linear)
[    0.900872] Mountpoint-cache hash table entries: 32768 (order: 6, 262144 bytes, linear)
[    0.900872] Last level iTLB entries: 4KB 64, 2MB 8, 4MB 8
[    0.900872] Last level dTLB entries: 4KB 64, 2MB 0, 4MB 0, 1GB 4
[    0.900872] Spectre V1 : Mitigation: usercopy/swapgs barriers and __user pointer sanitization
[    0.900872] Spectre V2 : Mitigation: Full generic retpoline
[    0.900872] Spectre V2 : Spectre v2 / SpectreRSB mitigation: Filling RSB on context switch
[    0.900872] Speculative Store Bypass: Vulnerable
[    0.900872] TAA: Vulnerable: Clear CPU buffers attempted, no microcode
[    0.900872] MDS: Vulnerable: Clear CPU buffers attempted, no microcode
[    0.900872] Freeing SMP alternatives memory: 28K
[    0.900872] smpboot: CPU0: Intel(R) Xeon(R) Platinum 8124M CPU @ 3.00GHz (family: 0x6, model: 0x55, stepping: 0x4)
[    0.900995] Performance Events: unsupported p6 CPU model 85 no PMU driver, software events only.
[    0.904919] rcu: Hierarchical SRCU implementation.
[    0.908818] NMI watchdog: Perf NMI watchdog permanently disabled
[    0.908991] smp: Bringing up secondary CPUs ...
[    0.912643] x86: Booting SMP configuration:
[    0.912877] .... node  #0, CPUs:      #1
[    0.844618] kvm-clock: cpu 1, msr 3401041, secondary cpu clock
[    0.914167] kvm-guest: stealtime: cpu 1, msr 40bc6d080
[    0.920970]  #2
[    0.844618] kvm-clock: cpu 2, msr 3401081, secondary cpu clock
[    0.921801] kvm-guest: stealtime: cpu 2, msr 40bcad080
[    0.928967]  #3
[    0.844618] kvm-clock: cpu 3, msr 34010c1, secondary cpu clock
[    0.933262] kvm-guest: stealtime: cpu 3, msr 40bced080
[    0.940964]  #4
[    0.844618] kvm-clock: cpu 4, msr 3401101, secondary cpu clock
[    0.941988] kvm-guest: stealtime: cpu 4, msr 40bd2d080
[    0.948936] MDS CPU bug present and SMT on, data leak possible. See https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/mds.html for more details.
[    0.952878] TAA CPU bug present and SMT on, data leak possible. See https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/tsx_async_abort.html for more details.
[    0.956985]  #5
[    0.844618] kvm-clock: cpu 5, msr 3401141, secondary cpu clock
[    0.957824] kvm-guest: stealtime: cpu 5, msr 40bd6d080
[    0.964968]  #6
[    0.844618] kvm-clock: cpu 6, msr 3401181, secondary cpu clock
[    0.965799] kvm-guest: stealtime: cpu 6, msr 40bdad080
[    0.972968]  #7
[    0.844618] kvm-clock: cpu 7, msr 34011c1, secondary cpu clock
[    0.973909] kvm-guest: stealtime: cpu 7, msr 40bded080
[    0.980902] smp: Brought up 1 node, 8 CPUs
[    0.984879] smpboot: Max logical packages: 1
[    0.988317] smpboot: Total of 8 processors activated (47999.96 BogoMIPS)
[    0.996908] node 0 deferred pages initialised in 8ms
[    1.001364] devtmpfs: initialized
[    1.004919] x86/mm: Memory block size: 128MB
[    1.009072] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
[    1.016884] futex hash table entries: 2048 (order: 5, 131072 bytes, linear)
[    1.021102] NET: Registered protocol family 16
[    1.024676] DMA: preallocated 2048 KiB GFP_KERNEL pool for atomic allocations
[    1.028882] DMA: preallocated 2048 KiB GFP_KERNEL|GFP_DMA pool for atomic allocations
[    1.032882] DMA: preallocated 2048 KiB GFP_KERNEL|GFP_DMA32 pool for atomic allocations
[    1.040896] audit: initializing netlink subsys (disabled)
[    1.044892] audit: type=2000 audit(1633306613.966:1): state=initialized audit_enabled=0 res=1
[    1.044982] thermal_sys: Registered thermal governor 'fair_share'
[    1.048879] thermal_sys: Registered thermal governor 'step_wise'
[    1.052876] thermal_sys: Registered thermal governor 'user_space'
[    1.056905] cpuidle: using governor ladder
[    1.064210] cpuidle: using governor menu
[    1.064983] ACPI: bus type PCI registered
[    1.068292] acpiphp: ACPI Hot Plug PCI Controller Driver version: 0.5
[    1.073017] PCI: Using configuration type 1 for base access
[    1.081215] Kprobes globally optimized
[    1.084907] HugeTLB registered 1.00 GiB page size, pre-allocated 0 pages
[    1.088878] HugeTLB registered 2.00 MiB page size, pre-allocated 0 pages
[    1.197871] DRBG: Continuing without Jitter RNG
[    1.202761] ACPI: Added _OSI(Module Device)
[    1.204879] ACPI: Added _OSI(Processor Device)
[    1.208470] ACPI: Added _OSI(3.0 _SCP Extensions)
[    1.212876] ACPI: Added _OSI(Processor Aggregator Device)
[    1.216869] ACPI: Added _OSI(Linux-Dell-Video)
[    1.220876] ACPI: Added _OSI(Linux-Lenovo-NV-HDMI-Audio)
[    1.224848] ACPI: Added _OSI(Linux-HPI-Hybrid-Graphics)
[    1.229622] ACPI: 3 ACPI AML tables successfully acquired and loaded
[    1.233808] ACPI: Interpreter enabled
[    1.236883] ACPI: (supports S0 S4 S5)
[    1.240135] ACPI: Using IOAPIC for interrupt routing
[    1.244895] PCI: Using host bridge windows from ACPI; if necessary, use "pci=nocrs" and report a bug
[    1.253186] ACPI: Enabled 16 GPEs in block 00 to 0F
[    1.259088] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-ff])
[    1.260885] acpi PNP0A03:00: _OSC: OS supports [ASPM ClockPM Segments MSI HPX-Type3]
[    1.268885] acpi PNP0A03:00: fail to add MMCONFIG information, can't access extended PCI configuration space under this bridge.
[    1.277149] acpiphp: Slot [3] registered
[    1.280519] acpiphp: Slot [4] registered
[    1.284897] acpiphp: Slot [5] registered
[    1.288255] acpiphp: Slot [6] registered
[    1.292899] acpiphp: Slot [7] registered
[    1.296238] acpiphp: Slot [8] registered
[    1.296896] acpiphp: Slot [9] registered
[    1.300251] acpiphp: Slot [10] registered
[    1.304895] acpiphp: Slot [11] registered
[    1.308291] acpiphp: Slot [12] registered
[    1.312898] acpiphp: Slot [13] registered
[    1.316298] acpiphp: Slot [14] registered
[    1.316897] acpiphp: Slot [15] registered
[    1.320875] acpiphp: Slot [16] registered
[    1.324899] acpiphp: Slot [17] registered
[    1.328299] acpiphp: Slot [18] registered
[    1.332896] acpiphp: Slot [19] registered
[    1.336296] acpiphp: Slot [20] registered
[    1.336900] acpiphp: Slot [21] registered
[    1.340876] acpiphp: Slot [22] registered
[    1.344897] acpiphp: Slot [23] registered
[    1.348296] acpiphp: Slot [24] registered
[    1.352895] acpiphp: Slot [25] registered
[    1.356268] acpiphp: Slot [26] registered
[    1.356895] acpiphp: Slot [27] registered
[    1.360875] acpiphp: Slot [28] registered
[    1.364898] acpiphp: Slot [29] registered
[    1.368337] acpiphp: Slot [30] registered
[    1.372899] acpiphp: Slot [31] registered
[    1.376311] PCI host bridge to bus 0000:00
[    1.380877] pci_bus 0000:00: root bus resource [io  0x0000-0x0cf7 window]
[    1.384879] pci_bus 0000:00: root bus resource [io  0x0d00-0xffff window]
[    1.388876] pci_bus 0000:00: root bus resource [mem 0x000a0000-0x000bffff window]
[    1.396876] pci_bus 0000:00: root bus resource [mem 0xc0000000-0xfebfffff window]
[    1.400876] pci_bus 0000:00: root bus resource [bus 00-ff]
[    1.404917] pci 0000:00:00.0: [8086:1237] type 00 class 0x060000
[    1.409357] pci 0000:00:01.0: [8086:7000] type 00 class 0x060100
[    1.416875] pci 0000:00:01.3: [8086:7113] type 00 class 0x000000
[    1.421833] pci 0000:00:01.3: quirk: [io  0xb000-0xb03f] claimed by PIIX4 ACPI
[    1.428895] pci 0000:00:01.3: quirk: [io  0xb100-0xb10f] claimed by PIIX4 SMB
[    1.432941] pci 0000:00:01.3: PIIX4 devres E PIO at fff0-ffff
[    1.436892] pci 0000:00:01.3: PIIX4 devres F MMIO at ffc00000-ffffffff
[    1.440894] pci 0000:00:01.3: PIIX4 devres G PIO at fff0-ffff
[    1.444895] pci 0000:00:01.3: PIIX4 devres H MMIO at ffc00000-ffffffff
[    1.448894] pci 0000:00:01.3: PIIX4 devres I PIO at fff0-ffff
[    1.452893] pci 0000:00:01.3: PIIX4 devres J PIO at fff0-ffff
[    1.456879] pci 0000:00:01.3: quirk_piix4_acpi+0x0/0x170 took 35156 usecs
[    1.461220] pci 0000:00:03.0: [1d0f:1111] type 00 class 0x030000
[    1.465274] pci 0000:00:03.0: reg 0x10: [mem 0xfe400000-0xfe7fffff pref]
[    1.474009] pci 0000:00:03.0: reg 0x30: [mem 0xfebd0000-0xfebdffff pref]
[    1.477204] pci 0000:00:04.0: [1d0f:8061] type 00 class 0x010802
[    1.482149] pci 0000:00:04.0: reg 0x10: [mem 0xfebf0000-0xfebf3fff]
[    1.492875] pci 0000:00:05.0: [1d0f:ec20] type 00 class 0x020000
[    1.497865] pci 0000:00:05.0: reg 0x10: [mem 0xfebf4000-0xfebf7fff]
[    1.502465] pci 0000:00:05.0: reg 0x18: [mem 0xfe800000-0xfe8fffff pref]
[    1.510345] pci 0000:00:05.0: reg 0x20: [mem 0xfebe0000-0xfebeffff]
[    1.515124] pci 0000:00:1f.0: [1d0f:8061] type 00 class 0x010802
[    1.521874] pci 0000:00:1f.0: reg 0x10: [mem 0xfebf8000-0xfebfbfff]
[    1.529712] ACPI: PCI Interrupt Link [LNKA] (IRQs 5 *10 11)
[    1.532983] ACPI: PCI Interrupt Link [LNKB] (IRQs 5 *10 11)
[    1.536978] ACPI: PCI Interrupt Link [LNKC] (IRQs 5 10 *11)
[    1.540975] ACPI: PCI Interrupt Link [LNKD] (IRQs 5 10 *11)
[    1.544927] ACPI: PCI Interrupt Link [LNKS] (IRQs *9)
[    1.552941] iommu: Default domain type: Translated 
[    1.556907] pci 0000:00:03.0: vgaarb: setting as boot VGA device
[    1.560872] pci 0000:00:03.0: vgaarb: VGA device added: decodes=io+mem,owns=io+mem,locks=none
[    1.560888] pci 0000:00:03.0: vgaarb: bridge control possible
[    1.564881] vgaarb: loaded
[    1.567832] EDAC MC: Ver: 3.0.0
[    1.569114] NetLabel: Initializing
[    1.572147] NetLabel:  domain hash size = 128
[    1.576887] NetLabel:  protocols = UNLABELED CIPSOv4 CALIPSO
[    1.580890] NetLabel:  unlabeled traffic allowed by default
[    1.584876] PCI: Using ACPI for IRQ routing
[    1.588953] hpet0: at MMIO 0xfed00000, IRQs 2, 8, 0, 0, 0, 0, 0, 0
[    1.592878] hpet0: 8 comparators, 32-bit 62.500000 MHz counter
[    1.696007] clocksource: Switched to clocksource kvm-clock
[    1.704197] VFS: Disk quotas dquot_6.6.0
[    1.707654] VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
[    1.712386] pnp: PnP ACPI init
[    1.715810] pnp: PnP ACPI: found 5 devices
[    1.725867] clocksource: acpi_pm: mask: 0xffffff max_cycles: 0xffffff, max_idle_ns: 2085701024 ns
[    1.733031] NET: Registered protocol family 2
[    1.737046] IP idents hash table entries: 262144 (order: 9, 2097152 bytes, linear)
[    1.745215] tcp_listen_portaddr_hash hash table entries: 8192 (order: 5, 131072 bytes, linear)
[    1.752248] TCP established hash table entries: 131072 (order: 8, 1048576 bytes, linear)
[    1.759261] TCP bind hash table entries: 65536 (order: 8, 1048576 bytes, linear)
[    1.766049] TCP: Hash tables configured (established 131072 bind 65536)
[    1.770896] UDP hash table entries: 8192 (order: 6, 262144 bytes, linear)
[    1.775699] UDP-Lite hash table entries: 8192 (order: 6, 262144 bytes, linear)
[    1.782273] NET: Registered protocol family 1
[    1.785882] pci_bus 0000:00: resource 4 [io  0x0000-0x0cf7 window]
[    1.790231] pci_bus 0000:00: resource 5 [io  0x0d00-0xffff window]
[    1.794621] pci_bus 0000:00: resource 6 [mem 0x000a0000-0x000bffff window]
[    1.799302] pci_bus 0000:00: resource 7 [mem 0xc0000000-0xfebfffff window]
[    1.804010] pci 0000:00:00.0: Limiting direct PCI/PCI transfers
[    1.808286] pci 0000:00:01.0: Activating ISA DMA hang workarounds
[    1.812750] pci 0000:00:03.0: Video device with shadowed ROM at [mem 0x000c0000-0x000dffff]
[    1.819700] PCI: CLS 0 bytes, default 64
[    1.823171] Trying to unpack rootfs image as initramfs...
[    2.485036] Freeing initrd memory: 7780K
[    2.488491] PCI-DMA: Using software bounce buffering for IO (SWIOTLB)
[    2.492925] software IO TLB: mapped [mem 0x00000000bbfe9000-0x00000000bffe9000] (64MB)
[    2.499695] clocksource: tsc: mask: 0xffffffffffffffff max_cycles: 0x2b3e43c8763, max_idle_ns: 440795360101 ns
[    2.507404] clocksource: Switched to clocksource tsc
[    2.511948] check: Scanning for low memory corruption every 60 seconds
[    2.519015] Initialise system trusted keyrings
[    2.522718] Key type blacklist registered
[    2.526228] workingset: timestamp_bits=36 max_order=22 bucket_order=0
[    2.531516] zbud: loaded
[    2.541298] Key type asymmetric registered
[    2.544769] Asymmetric key parser 'x509' registered
[    2.548623] Block layer SCSI generic (bsg) driver version 0.4 loaded (major 251)
[    2.555391] crc32: CRC_LE_BITS = 64, CRC_BE BITS = 64
[    2.559258] crc32: self tests passed, processed 225944 bytes in 121526 nsec
[    2.564011] crc32c: CRC_LE_BITS = 64
[    2.567270] crc32c: self tests passed, processed 225944 bytes in 60724 nsec
[    2.581071] crc32_combine: 8373 self tests passed
[    2.593937] crc32c_combine: 8373 self tests passed
[    2.598242] Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
[    2.602813] 00:04: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is a 16550A
[    2.610296] nvme nvme0: pci function 0000:00:04.0
[    2.614212] nvme nvme1: pci function 0000:00:1f.0
[    2.614412] PCI Interrupt Link [LNKD] enabled at IRQ 11
[    2.617997] ixgbevf: Intel(R) 10 Gigabit PCI Express Virtual Function Network Driver
[    2.622153] PCI Interrupt Link [LNKC] enabled at IRQ 10
[    2.628747] nvme nvme0: 2/0/0 default/read/poll queues
[    2.628904] ixgbevf: Copyright (c) 2009 - 2018 Intel Corporation.
[    2.632969]  nvme0n1: p1 p128
[    2.638368] nvme nvme1: 2/0/0 default/read/poll queues
[    2.641947] i8042: PNP: PS/2 Controller [PNP0303:KBD,PNP0f13:MOU] at 0x60,0x64 irq 1,12
[    2.655787] i8042: Warning: Keylock active
[    2.660190] serio: i8042 KBD port at 0x60,0x64 irq 1
[    2.664063] serio: i8042 AUX port at 0x60,0x64 irq 12
[    2.668066] rtc_cmos 00:00: RTC can wake from S4
[    2.671937]  nvme1n1: p1
[    2.672586] rtc_cmos 00:00: registered as rtc0
[    2.678731] rtc_cmos 00:00: setting system clock to 2021-10-04T00:16:56 UTC (1633306616)
[    2.685456] rtc_cmos 00:00: alarms up to one day, 114 bytes nvram
[    2.689787] intel_pstate: P-states controlled by the platform
[    2.693910] hid: raw HID events driver (C) Jiri Kosina
[    2.697862] NET: Registered protocol family 17
[    2.702129] IPI shorthand broadcast: enabled
[    2.705726] sched_clock: Marking stable (1865090351, 840618358)->(2834464274, -128755565)
[    2.712500] registered taskstats version 1
[    2.715969] Loading compiled-in X.509 certificates
[    2.720375] Loaded X.509 cert 'Build time autogenerated kernel key: e7ea7d38767a6db571542ec00d2a057a1a3ce074'
[    2.727867] zswap: loaded using pool lzo/zbud
[    2.731544] ima: No TPM chip found, activating TPM-bypass!
[    2.735559] ima: Allocated hash algorithm: sha1
[    2.739167] ima: No architecture policies found
[    2.743283] TAP version 14
[    2.746175] 1..0
[    2.749998] Freeing unused decrypted memory: 2036K
[    2.754507] Freeing unused kernel image (initmem) memory: 2252K
[    2.788895] Write protecting the kernel read-only data: 16384k
[    2.793725] Freeing unused kernel image (text/rodata gap) memory: 2040K
[    2.798485] Freeing unused kernel image (rodata/data gap) memory: 504K
[    2.803033] Run /init as init process
[32;1m>>[0;39m[1m Genkernel 4.2.3 (2021-10-04 00:10:58 UTC). Linux kernel 5.10.61-gentoo-x86_64 [0;39m
[32;1m>>[0;39m[1m Activating udev ... [0;39m
[32;1m>>[0;39m[1m Determining root device (trying UUID=4635af18-d23d-4770-a867-3d3d32b283da) ...
[32;1m>>[0;39m[1m Root device detected as /dev/nvme1n1p1! [0;39m
[32;1m>>[0;39m[1m Mounting /dev/nvme1n1p1 as root ... [0;39m
[32;1m>>[0;39m[1m Using mount -t ext4 -o ro /dev/nvme1n1p1 /newroot [0;39m
[32;1m>>[0;39m[1m Restoring console log level (7) ... [0;39m
[32;1m>>[0;39m[1m Switching to real root: switch_root /newroot /sbin/init  [0;39m

INIT: version 2.99 booting

[    3.305329] random: fast init done

   [1m[32mOpenRC [1m[36m0.42.1[m is starting up [1m[34mGentoo Linux (x86_64)[m

 [1m[32m*[m /proc is already mounted[K
 [1m[32m*[m /run/openrc: creating directory[K
 [1m[32m*[m /run/lock: creating directory[K
 [1m[32m*[m /run/lock: correcting owner[K
 [1m[32m*[m Caching service dependencies[K ...
 [ ok ]
 [1m[32m*[m Mounting security filesystem[K ...
 [ ok ]
 [1m[32m*[m Mounting debug filesystem[K ...
 [ ok ]
 [1m[32m*[m Mounting SELinux filesystem[K ...
 [ ok ]
 [1m[32m*[m Mounting persistent storage (pstore) filesystem[K ...
 [ ok ]
 [1m[32m*[m Mounting cgroup filesystem[K ...
 [ ok ]
 [1m[32m*[m Remounting devtmpfs on /dev[K ...
 [ ok ]
 [1m[32m*[m Mounting /dev/mqueue[K ...
 [ ok ]
 [1m[32m*[m Creating list of required static device nodes for the current kernel[K ...
 [ ok ]
 [1m[32m*[m Setting up tmpfiles.d entries for /dev[K ...
 [ ok ]
 [1m[32m*[m Starting udev[K ...
 [ ok ]
 [1m[32m*[m Generating a rule to create a /dev/root symlink[K ...
 [ ok ]
 [1m[32m*[m Populating /dev with existing devices through uevents[K ...
 [ ok ]
 [1m[32m*[m Loading kernel modules[K ...
insmod /lib/modules/5.10.61-gentoo-x86_64/net/ena.ko 
modprobe: ERROR: could not insert 'ena': Module already in kernel
 [ ok ]
 [1m[32m*[m Setting system clock using the hardware clock [UTC][K ...
 [ ok ]
 [1m[33m*[m The binfmt-misc module needs to be loaded by the modules service or built in.[K
 [1m[32m*[m Mounting misc binary format filesystem[K ...
 [ ok ]
 [1m[32m*[m Loading custom binary format handlers[K ...
 [ ok ]
 [1m[32m*[m Checking local filesystems [K ...
aux-root: clean, 309511/1310720 files, 2237924/5242363 blocks
 [ ok ]
 [1m[32m*[m Remounting root filesystem read/write[K ...
 [ ok ]
 [1m[32m*[m Remounting filesystems[K ...
 [ ok ]
 [1m[32m*[m Updating /etc/mtab[K ...
 [1m[32m*[m Creating mtab symbolic link[K
 [ ok ]
 [1m[32m*[m Activating swap devices[K ...
 [ ok ]
 [1m[32m*[m Mounting local filesystems[K ...
 [ ok ]
 [1m[32m*[m Configuring kernel parameters[K ...
 [ ok ]
 [1m[32m*[m Creating user login records[K ...
 [ ok ]
 [1m[32m*[m Wiping /tmp directory[K ...
 [ ok ]
 [1m[32m*[m Bringing up interface eth0[K
 [1m[32m*[m   Caching network module dependencies[K
 [1m[33m*[m   config_eth0 not specified; defaulting to DHCP[K
 [1m[32m*[m   dhcp[K ...
 [1m[31m*[m     nothing provides `dhcp'[K
 [ !! ]
 [1m[31m*[m ERROR: net.eth0 failed to start[K
 [1m[31m*[m ERROR: cannot start ec2-init as net.eth0 would not start[K
 [1m[32m*[m Setting hostname to localhost [K ...
 [ ok ]
 [1m[32m*[m Bringing up network interface lo[K ...
 [ ok ]
 [1m[32m*[m Setting terminal encoding [UTF-8][K ...
 [ ok ]
 [1m[32m*[m Saving key mapping[K ...
 [ ok ]
 [1m[32m*[m Saving terminal encoding[K ...
 [ ok ]
 [1m[32m*[m Setting up tmpfiles.d entries[K ...
 [ ok ]

INIT: Entering runlevel: 3

 [1m[32m*[m Bringing up interface eth0[K
 [1m[33m*[m   config_eth0 not specified; defaulting to DHCP[K
 [1m[32m*[m   dhcp[K ...
 [1m[31m*[m     nothing provides `dhcp'[K
 [ !! ]
 [1m[31m*[m ERROR: net.eth0 failed to start[K
 [1m[31m*[m ERROR: cannot start ec2-init as net.eth0 would not start[K
 [1m[31m*[m ERROR: cannot start netmount as net.eth0 would not start[K
ssh-keygen: generating new host keys: RSA DSA ECDSA ED25519 
 [1m[32m*[m Starting sshd[K ...
 [ ok ]
 [1m[32m*[m Starting local[K ...
 [ ok ]



This is localhost.unknown_domain (Linux x86_64 5.10.61-gentoo-x86_64) 00:17:04
```

I guess dhcp client by default is not installed in fresh gentoo.
I will test the fix and will release new version.

**Comment by sormy** [comment]
I have pushed a fix in v1.1.4. Please verify it fixes the issue for you.

**Comment by MrKnarfy** [comment]
Works perfectly! Thanks!
